pages/common/subfinder.md

```diff
@@ -12,7 +12,7 @@
 
 `subfinder --silent -d {{example.com}}`
 
-- Use bruteforcing to find subdomains:
+- Use brute force to find subdomains:
```

Comment:
I'm not sure if `brute force` would be taken a bit too literal in this context. (Altough it sounds really funny 😄). How about:
```suggestion
- Use a brute-force attack to find subdomains:
```